config/environments/development.rb

```diff
@@ -86,14 +86,6 @@
   config.slimmer.asset_host = ENV['STATIC_DEV'] || Plek.find('static')
 
   if ENV['SHOW_PRODUCTION_IMAGES']
-    orig_host = config.asset_host
-    config.asset_host = Proc.new do |source|
-      local_file = File.join(Whitehall.clean_uploads_root, source.sub('/government/uploads', ''))
-      if !File.exist?(local_file) && source =~ %r{system/uploads}
-        "https://assets.publishing.service.gov.uk"
-      else
-        orig_host
-      end
-    end
+    config.asset_host = 'https://assets.publishing.service.gov.uk'
```

Comment:
Should we be using Plek to get this?

Reply:
Plek to get the production assets host?

Reply:
Hmm good point, I guess this isn't set as the same in dev environments.